<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>177</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Playground</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>573</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNewProject"/>
    <addaction name="actionOpenProject"/>
    <addaction name="actionSaveProject"/>
    <addaction name="actionProjectSetting"/>
    <addaction name="actionCloseProject"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Settings">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionSetting"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuScene">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>&amp;Scene</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuObject">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>&amp;Object</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuComponent">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>&amp;Component</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuViewport">
    <property name="title">
     <string>Viewport</string>
    </property>
    <addaction name="actionCreateNewViewport"/>
   </widget>
   <widget class="QMenu" name="menuAssets">
    <property name="title">
     <string>Assets</string>
    </property>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuAssets"/>
   <addaction name="menuScene"/>
   <addaction name="menuViewport"/>
   <addaction name="menuObject"/>
   <addaction name="menuComponent"/>
   <addaction name="menu_Settings"/>
   <addaction name="menu_Help"/>
  </widget>
  <action name="actionSetting">
   <property name="icon">
    <iconset resource="../playground.qrc">
     <normaloff>:/images/preferences-system.png</normaloff>:/images/preferences-system.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Settings</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="../playground.qrc">
     <normaloff>:/images/preferences-system.png</normaloff>:/images/preferences-system.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Settings</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../playground.qrc">
     <normaloff>:/images/application-exit.png</normaloff>:/images/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionNewProject">
   <property name="icon">
    <iconset resource="../playground.qrc">
     <normaloff>:/images/game-new.png</normaloff>:/images/game-new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New project</string>
   </property>
  </action>
  <action name="actionOpenProject">
   <property name="icon">
    <iconset resource="../playground.qrc">
     <normaloff>:/images/game-open.png</normaloff>:/images/game-open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open project</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionCloseProject">
   <property name="text">
    <string>&amp;Close project</string>
   </property>
  </action>
  <action name="actionSaveProject">
   <property name="icon">
    <iconset resource="../playground.qrc">
     <normaloff>:/images/game-save.png</normaloff>:/images/game-save.png</iconset>
   </property>
   <property name="text">
    <string>Save project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionProjectSetting">
   <property name="icon">
    <iconset resource="../playground.qrc">
     <normaloff>:/images/game-configure.png</normaloff>:/images/game-configure.png</iconset>
   </property>
   <property name="text">
    <string>Project setting</string>
   </property>
  </action>
  <action name="actionCreateNewViewport">
   <property name="text">
    <string>Create new</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../playground.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionNewProject</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>newProject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>quit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout_Qt</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>aboutQt()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSetting</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>showModulesSettingsDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>newProject()</slot>
  <slot>quit()</slot>
  <slot>aboutQt()</slot>
  <slot>showModulesSettingsDialog()</slot>
 </slots>
</ui>
